src/goMain.ts

```diff
@@ -56,6 +56,7 @@ export function activate(ctx: vscode.ExtensionContext): void {
 
 	let onChange = _.debounce(maybeAutorunTestsOnChange, 200);
 	watcher.onDidChange(onChange);
+	watcher.onDidChange(updatePinnedTestLocation);
```

Comment:
it seems like we should debounce this as well?

Reply:
Yeah good idea